### PR TITLE
Remove deprecated parameters of `#[AsDocumentListener]`

### PR DESCRIPTION
**Pull request by GromNaN**
Follows https://github.com/doctrine/DoctrineMongoDBBundle/pull/825

### CODE REVIEW

File: UPGRADE-5.0.md [hunk 1]
```diff
@@ -42,6 +42,8 @@ Deprecated options have been removed:
 
 * Remove `Doctrine\Bundle\MongoDBBundle\EventSubscriber\EventSubscriberInterface`.
   Use the `#[AsDocumentListener]` attribute instead.
+* Remove parameters `$method` and `$lazy` of `#[AsDocumentListener]`, they were
+  are not used.
```
Comment (franmomu) [inline UPGRADE-5.0.md]:
```suggestion
* Remove parameters `$method` and `$lazy` of `#[AsDocumentListener]`, they were
 not used.
```
I guess `were` is the correct one

**Review by franmomu — APPROVED** [review]

File: UPGRADE-5.0.md (hunk 1 above)
Comment (GromNaN) [reply UPGRADE-5.0.md]: Fixed. Thanks.

**Comment by franmomu** [comment]
thanks @GromNaN!

**Comment by mustanggb** [comment]
Docs need updating: [events.rst](https://github.com/doctrine/DoctrineMongoDBBundle/blob/5.0.x/docs/events.rst?plain=1#L27)

**Comment by malarzm** [comment]
@mustanggb thanks for noticing! Mind opening a PR?

**Pull request merged**